webapp/channels/src/i18n/en.json

```diff
@@ -3972,19 +3970,19 @@
   "learn_more_about_trial.modal.useSsoDescription": "Sign on quickly and easily with our SSO feature that works with OpenID, SAML, Google, and O365.",
   "learn_more_about_trial.modal.useSsoTitle": "Use SSO (with OpenID, SAML, Google, O365)",
   "learn_more_trial_modal_step.learnMoreAboutFeature": "Learn more about this feature.",
-  "learn_more_trial_modal.contact_sales": "Contact Sales",
+  "learn_more_trial_modal.contact_sales": "Contact sales",
   "learn_more_trial_modal.pretitle": "With Enterprise, you can...",
   "leave_private_channel_modal.leave": "Yes, leave channel",
   "leave_private_channel_modal.message": "Are you sure you wish to leave the private channel {channel}? You must be re-invited in order to re-join this channel in the future.",
   "leave_private_channel_modal.title": "Leave Private Channel {channel}",
   "leave_public_channel_modal.message": "Are you sure you wish to leave the channel {channel}? You can re-join this channel in the future if you change your mind.",
   "leave_public_channel_modal.title": "Leave Channel {channel}",
-  "leave_team_modal_guest_only_private.desc": "**You will be removed from {num_of_private_channels} private { num_of_private_channels,plural,one {channel} other {channels}} on this team.** You won't be able to rejoin it without an invitation from another team member. Are you sure?",
-  "leave_team_modal_guest_only_public.desc": "**You will be removed from {num_of_public_channels} public { num_of_public_channels,plural,one {channel} other {channels}} on this team.** You won't be able to rejoin it without an invitation from another team member. Are you sure?",
-  "leave_team_modal_guest.desc": "**You will be removed from {num_of_public_channels} public { num_of_public_channels,plural,one {channel} other {channels}} and {num_of_private_channels} private { num_of_private_channels,plural,one {channel} other {channels}} on this team.** You won't be able to rejoin it without an invitation from another team member. Are you sure?",
-  "leave_team_modal_private.desc": "**You will be removed from {num_of_private_channels} private { num_of_private_channels,plural,one {channel} other {channels} } on this team.** If the team is private you won't be able to rejoin it the team without an invitation from another team member. Are you sure?",
+  "leave_team_modal_guest_only_private.desc": "** You will be removed from {num_of_private_channels} private { num_of_private_channels,plural,one {channel} other {channels}} on this team.** You won't be able to rejoin it without an invitation from another team member. Are you sure?",
```

Comment:
The extra spaces added after ``**`` would invalidate the bold formatting. Intended?